substrate/frame/staking-async/ahm-test/src/ah/mock.rs

```diff
@@ -319,8 +409,10 @@ parameter_types! {
 	pub static BondingDuration: u32 = 3;
 	pub static SlashDeferredDuration: u32 = 2;
 	pub static SessionsPerEra: u32 = 6;
-	pub static PlanningEraOffset: u32 = 2;
+	// Begin election as soon as a new era starts.
+	pub static PlanningEraOffset: u32 = 6;
```

Comment:
skeptical, have to check why this has to be 6, I would have thought 0 is the right new default now.

Reply:
ah sorry, we subtract 6 from this, so this leads to the 0 that I was thinking of.